### ISSUE TITLE
Continuity Camera / Sketch doesn't work in Typora

### ISSUE THREAD

**Issue by lisc55** [typora/typora-issues#4673]
- [x] Searched existing issues to avoid creating duplicates.

- [x] Confirmed that it can be reproduced in built-in themes without customized css.

   If it only exists in 3rd party themes or css, you can still report it, but please attach the theme target or the css file. We may not "fix" it, if it is caused by 3rd party themes or css styles that we cannot support.

- [x] Searched http://support.typora.io/

----

**Describe the bug**
"Insert from iPhone" doesn't work. After I take the photo or finish the sketch, Typora has no response. Meanwhile, they work in other apps like TextEdit or PowerPoint. 

**To Reproduce**
1. Click "Format" -- "Insert from iPhone" --- "Take Photo"
2. "Take a photo with xxx" pops out and iPhone's camera is on
3. After taking the photo, "Take a photo with xxx" disappear but nothing appears. 

**Expected behavior**
The photo should appear in Typora. 

**Screenshots / Screencasts**
<img width="1680" alt="截屏2021-09-22 下午2 01 40" src="https://user-images.githubusercontent.com/21216965/134292857-aefb9f0e-a4bb-439c-93d1-247ee2a339bc.png">
<img width="1792" alt="截屏2021-09-22 下午2 01 57" src="https://user-images.githubusercontent.com/21216965/134292859-19a361be-9cb0-41b5-be65-07fb8fef4a69.png">
<img width="1792" alt="截屏2021-09-22 下午2 02 16" src="https://user-images.githubusercontent.com/21216965/134292860-662ece75-6dbf-41e8-a3ad-de4599776258.png">
<img width="1680" alt="截屏2021-09-22 下午2 05 45" src="https://user-images.githubusercontent.com/21216965/134292853-07dd0cfa-8c3b-47b2-ab4f-dfe8ebd4472d.png">

**Desktop (please complete the following information):**
 - OS: macOS Catalina 10.15.7
 - iOS on iPhone: 14.7.1

**Typora Version**
0.11.5

**Additional context**
After failed, the buttons "Take photo", "Scan documents", "Add sketch" are grayed out as the last screenshot shows. But meanwhile these buttons are available in other apps (eg. TextEdit, MS PowerPoint), and they function normally in these apps.

**Comment by abnerlee** [comment]
They work in macOS 11 in my test..

**Comment by lisc55** [comment]
I think I've been using the default image settings. 

<img width="832" alt="image" src="https://user-images.githubusercontent.com/21216965/135713755-aed32002-2258-42ff-b96b-71852d48c609.png">

**Comment by abnerlee** [comment]
Close since we still cannot reproduce it, and also not reported by other users.

**Comment by winny1001** [comment]
Typora: 1.4.4
macOS Monterey 12.6 (macbook pro M1 pro)
iPadOS 15.7 (iPad 6)
iOS 14.0.1 (iPhone 6s)

I still have the same bug.
Even after I uninstalled and reinstalled Typora, there was still the same problem.
All settings are default. The theme is build-in Night.
I tried other build-in themes, change language(I use Chinese), change image insert settings, I also installed Typora 1.4.1-dev, but none of them can fix this bug.

I found that the images and sketchs will be saved in /Library/Application Support/typora-user-images
but they can't appear in Typora automatically.
